src/content/appList.md

```diff
@@ -22,11 +15,18 @@ en:
               "description": "Writing NFT is the NFT of a piece of writing, literally. It can refer to articles or any type of textual content with or without illustrations."
             },
             {
-              "image": "/uploads/apps/matters.svg",
-              "title": "Matters",
-              "url": "https://matters.news",
+              "image": "/uploads/apps/depub-space.svg",
+              "title": "depub.space",
+              "url": "https://depub.space",
+              "isOfficial": true,
+              "description": "A Web3 decentralized Twitter that requires only a small amount of LIKE to directly “DePub” (decentralized-publish) text and images to blockchain and IPFS."
+            },
+            {
+              "image": "/uploads/apps/nft-portal.png",
+              "title": "NFT Portal",
+              "url": "https://app.like.co",
```

Comment:
Now the whitelist is up, should we throw ppl into `https://app.like.co/nft/`? Now it only shows iscn which it confusing
Additional redirect need to be made in app-like-co though